development/mattermost/nautobot_bootstrap.py

```diff
@@ -48,7 +49,7 @@
 with contextlib.suppress(ObjectDoesNotExist):
     admin = User.objects.get(name="admin")
     ChatOpsAccountLink.objects.update_or_create(
-        user_id="jactwicuqb8bu8pau8mgjydzeo",
```

Comment:
Why did this change?

Reply:
To match the Mattermost SQL dump. Mattermost integration didn't work without this change